jobs/base_jobs_test.py

```diff
@@ -71,7 +71,8 @@ class FooJob(python_utils.with_metaclass(MockJobMetaclass)):
             def __init__(self):
                 pass
 
-        del FooJob # NOTE: Deletes the variable, not the class.
+        # NOTE: Deletes the variable, not the class.
```

Comment:
This change is done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.